vmcnet/train/default_config.py

```diff
@@ -114,20 +114,28 @@ def get_default_model_config() -> Dict:
         "include_ee_norm": True,
     }
 
+    # use the self attention layer (instead of the multi-head attention layer)
+    num_heads = 1
+    use_transformer = True
```

Comment:
Actually could we set the default to False to preserve existing code-base behavior, and we can manually set it to True at run-time when we're testing the self-attention?

Reply:
sg